WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -78,8 +78,7 @@ private extension PaymentCaptureOrchestrator {
                                     captureResult: Result<PaymentIntent, Error>,
                                     onCompletion: @escaping (Result<CardPresentReceiptParameters, Error>) -> Void) {
         switch captureResult {
-        case .failure:
-            let error = CardReaderServiceError.paymentCapture()
+        case .failure(let error):
```

Comment:
@ctarda was there a reason to replace the received error with a new one? This was erasing all the details and logging an `internalServiceError` instead

Reply:
Not that I recall. Go figure 🤷🏼